custom/customize.sh

```diff
@@ -29,6 +29,8 @@ fi
 
 if [ -f custom/css/$APP.css ]; then
   cp custom/css/$APP.css src/css/themeoverrides.css
+  mkdir -p public/src/css/
```

Comment:
is this because our setup for `publicPath` in webpack config is off?

Reply:
i dont know webpack that well, it looks like we want themeoverrides.css to be passed through as a separate file

Reply:
do you have a suggestion on how this should be handled?

Reply:
Honestly I think the issue discuribed in PXP-11107 should be fixed by https://github.com/uc-cdis/data-portal/pull/1402/files#diff-997f6766dcd75e3c8f1940f37f581261863b8c581e1a7e21e12568e5504cbf3cR31
With that being side, the loading logic for local css in portal is still a bit off: no matter what the `basename` value is, the path to local dev server css should always be `https://localhost:9443/src/css/themeoverrides.css`. This gliche should only affect you if you are working with an env whose portal has a customized `basename` value like HEAL. But for envs whose `basename == '/'`, local dev server dev loading should be working after that fix

Reply:
This ^ was merged on 09/14, I tried using a local gitops.css on 09/20 using master branch and couldn't get it to work

Reply:
hmmm can you try again now or let me know which env are you using? I'll try to replicate it

Reply:
its not working for anything that i have tested locally, currently testing https://ibdgc.datacommons.io/dev.html/, where is the file being served from? from what i see webpack does not expose any files not in the public directory so unless you place the file there its not being loaded locally

Reply:
hmm I think you are right, I must be testing something wrong last night, so this looks good
then can you update https://github.com/uc-cdis/data-portal/blob/master/boot.js#L31 so that it uses `https://localhost:9443/src/css/themeoverrides.css` only? It shouldn't be changing base on basenames